README.rst

```diff
@@ -39,7 +39,7 @@ For detailed information on how to create an issue in the documentation
 repository, please refer to `the "How To Create An Issue" guide
 <CREATING_ISSUES.rst>`_. 
 
-This process overrides **all** previous practices, and aims to formalise and
+This process overrides **all** previous practices and aims to formalize and
```

Comment:
Thanks for the PR @karna12. however, the initial comma can stay. Can you make that one change? The other change, from British to US English is great. Thanks for catching that.